scripts/build.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -e
+
+REPO=${REPO:-luxas/kubeadm-installer}
+KUBEADM_VERSION=${KUBEADM_VERSION:-v1.6.0-alpha.0.2074+a092d8e0f95f52}
+KUBEADM_REVISION=${KUBEADM_REVISION:-0}
+TAG_LATEST=${TAG_LATEST:-1}
+PUSH=${PUSH:-0}
+
+
+set -u
+
+# Workaround since Docker does not support SEMVER for tags
+# Especially the '+'
+#
+# https://github.com/docker/distribution/pull/1202
+KUBEADM_VERSION=${KUBEADM_VERSION/+/-g}
+
+
+PROJECT_DIR="$(git rev-parse --show-toplevel)"
+pushd ${PROJECT_DIR}
+docker build \
```

Comment:
why not just `docker build -t ${REPO}:${KUBEADM_VERSION}.${KUBEADM_REVISION} ${PROJECT_DIR}`?